elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -1071,18 +1045,45 @@ class PfeNavigation extends PFElement {
   }
 
   /**
-   * Translate strings based on object defined in constructor
+   * Translate strings based on passed in object
    */
   _translateStrings() {
-    //translate mobile menu button
-    this._mobileToggleText.textContent = this._navTranslations[this._lang].menu;
+    if (this._navTranslations) {
+      //translate mobile menu button
+      //will not update if mobile-menu-text attr is used
+      if (!this.mobileButtonTranslation) {
+        this._mobileButton.textContent = this._navTranslations[this._lang].menu;
+      }
 
-    //translate search string if used
-    if (this._searchToggle) {
-      this._searchToggleText.textContent = this._navTranslations[this._lang].search;
+      //translate search string if used
+      //will not update if search-button-text is used
+      if (this._searchToggle && !this.searchButtonTranslation) {
+        this._searchButtonText.textContent = this._navTranslations[this._lang].search;
+      }
     }
   }
 
+  /**
+   * Translate mobile menu button string
+   */
+  _updateMobileMenuText() {
+    this._mobileButton.textContent = this.mobileButtonTranslation;
+  }
+
+  /**
+   * Translate search button string
+   */
+  _updateSearchButtonText() {
+    this._searchButtonText.textContent = this.searchButtonTranslation;
+  }
+
+  /**
+   * Translate login string
+   */
+  // _updateLoginText() {
```

Comment:
What's up with this commented out fella

Reply:
Ah so I held off on including that until I figure out a workaround. This is to update the translation via attribute but the node isn't available when this runs..